pkgs/by-name/pi/pigpio/package.nix

```
@@ -0,0 +1,35 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  python3,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "pigpio";
+  version = "79";
+
+  src = fetchFromGitHub {
+    owner = "joan2937";
+    repo = pname;
+    rev = "refs/tags/v${version}";
+    hash = "sha256-Z+SwUlBbtWtnbjTe0IghR3gIKS43ZziN0amYtmXy7HE=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+  ];
+
+  postInstall = ''
+    install -Dm444 "$cmakeDir"/pigpio.py "$out/${python3.sitePackages}"/pigpio.py
+  '';
+
+  meta = with lib; {
+    description = "A C library for the Raspberry which allows control of the General Purpose Input Outputs (GPIO)";
```

https://github.com/NixOS/nixpkgs/tree/master/pkgs#meta-attributes

```suggestion
    description = "Raspberry library to control the General Purpose Input Outputs (GPIO)";
```